docs/ext/commands/commands.rst

```diff
@@ -1176,3 +1176,72 @@ For example, to block all DMs we could do the following:
     Be careful on how you write your global checks, as it could also lock you out of your own bot.
 
 .. need a note on global check once here I think
+
+
+Hybrid Commands
+---------------
+
+.. versionadded:: 2.0
+
+:class:`.commands.HybridCommand` is a command that can be invoked as both a text and a slash command.
+This allows you to define a command as both slash and text command without writing separate code for
+both counterparts.
+
+One of the goals of hybrid commands is to also allow you to easily transit from text commands to
```

Comment:
Not particularly. The goal is just to allow you to share functionality seamlessly.